Rx.xcodeproj/project.pbxproj

```diff
@@ -3893,6 +3893,7 @@
 		A2897D50225CA1E7004EA481 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				BUILD_LIBRARY_FOR_DISTRIBUTION = NO;
```

Comment:
Let's go with way of elimination. 

3 and 4 don't seem likely as both are manual and require a lot of maintenance. 

2 is easy, and also means "if you want this for your own use-case, you can achieve it manually using steps we can publish, etc."

1 is tricky. Changing the public API service just because of an Apple bug isn't a great idea today, at least. I'm wondering if we can mark these specific ones with deprecation availability of something, and if that would affect subclasses. Mind checking that?

Reply:
> I'm wondering if we can mark these specific ones with deprecation availability of something, and if that would affect subclasses. Mind checking that?

I'm not quite sure I understand what that means?
Do you mean can we annotate the code with some attribute or something, so it will start throwing compiler warnings in Xcode 12 so we don't forget to come back and fix it?

Reply:
The reason is to provide people from directly using the initializer on these types.

Reply:
What are your thoughts on this? @borland 

Based on the SR comment, the real fix might be right around the corner.